File: 1-easy/problems/2960-count-tested-devices-after-test-operations/solution.ts
function countTestedDevices(batteryPercentages: any): boolean | number | string | any {
  """
  :type batteryPercentages: List[int]
  :rtype: int
  """
  result = 0
  for (x in batteryPercentages) {
      if (x > result) {
          result += 1
  return result;
}

export { countTestedDevices };

File: 1-easy/problems/2960-count-tested-devices-after-test-operations/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def countTestedDevices(self, batteryPercentages):
        """
        :type batteryPercentages: List[int]
        :rtype: int
        """
        result = 0
        for x in batteryPercentages:
            if x > result:
                result += 1
        return result
